app.rb

```diff
@@ -0,0 +1,73 @@
+require_relative 'lib/person/student'
+require_relative 'lib/person/teacher'
+require_relative 'lib/book'
+require_relative 'lib/rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.empty?
+      print 'There are no registered books'
+    else
+      print 'Books:'
+      @books.each do |book|
+        print "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      print 'There are no registered people'
+    else
+      print 'People:'
+      @people.each do |person|
+        print "Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
```

Comment:
- [ ] Same here every time I attempted to list all the people, the application shut down :+1: